.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 120
+
+[{providers, services}/*/node_modules/**]
+insert_final_newline = false
```

Comment:
why is this needed?

Reply:
Peer deependant modules might use asm/wsm/wtc where this behaviour is not popular or required. Also we shouldn't assume on other peoples code (: